pkg/sources/github/github.go

```diff
@@ -894,11 +894,8 @@ func (s *Source) addUserGistsToCache(ctx context.Context, user string) error {
 	gistOpts := &github.GistListOptions{}
 	logger := s.log.WithValues("user", user)
 	for {
-		gists, res, err := s.apiClient.Gists.List(ctx, user, gistOpts)
-		if err == nil {
-			res.Body.Close()
-		}
```

Comment:
This shouldn't be necessary because go-github closes the response body.

https://github.com/google/go-github/blob/062b6119e47d73486fe40193262708332f1dc9be/github/github.go#L902-L907